### PR TITLE
UISMRCCOMP-1: Move common components from ui-quick-marc.

### PR DESCRIPTION
**Pull request by Dmytro-Melnyshyn**
## Purpose
Move common components from `ui-quick-marc`.

## Issue
[UISMRCCOMP-1](https://issues.folio.org/browse/UISMRCCOMP-1)

### CODE REVIEW

**Review by BogdanDenis — APPROVED** [review]
Looks good!

File: lib/MarcView/MarcContent/MarcContent.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+};
+
+MarcContent.propTypes = propTypes;
+
+MarcContent.defaultProps = {
+  isPrint: false,
+  tenantId: '',
+};
+
+export default MarcContent;
```
Comment (BogdanDenis) [inline lib/MarcView/MarcContent/MarcContent.js]: Let's use named exports in this modules to make sure we don't rename components when importing

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: lib/MarcView/MarcContent/MarcContent.js (hunk 1 above)
Comment (Dmytro-Melnyshyn) [reply lib/MarcView/MarcContent/MarcContent.js]: Thanks, fixed.

File: package.json [hunk 2]
```diff
@@ -43,15 +43,14 @@
     "react-intl": "^6.4.4",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "react-query": "^3.39.3",
-    "regenerator-runtime": "^0.13.3"
```
Comment (BogdanDenis) [inline package.json]: I think we still need this to transpile async functions

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**